src/sentry/grouping/api.py

```diff
@@ -47,7 +47,8 @@
 from sentry.utils.hashlib import md5_text
 
 if TYPE_CHECKING:
-    from sentry.grouping.fingerprinting import FingerprintingRules, FingerprintRuleJSON
+    from sentry.grouping.fingerprinting import FingerprintingRules
+    from sentry.grouping.fingerprinting.rules import FingerprintRuleJSON
```

Comment:
nit: imo this import reads bad, probably because `.rules` feels ambiguous when you have FingerprintingRules and FingerprintRuleJSON. is there a better way to structure where all the rules live in a single file?